content/articles/domain-privacy-after-gdpr.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: Domain privacy after GDPR
+excerpt: An A record maps a domain name to the IP address (IPv4) of the computer hosting the domain.
```

Comment:
Please update this excerpt.

Reply:
Oops, copy/pasta mistake. Thanks for catching that.